### PR TITLE
Added card component on doc page

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: Doc.html/cards.html [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+          <h2 class="drawer-title">Components</h2>
+
+          <ul>
+            <li class="list-item" onclick="handleListItemClick()">Alerts</li>
+            <li class="list-item" onclick="handleListItemClick()">Avatar</li>
+            <li class="list-item" onclick="handleListItemClick()">Badge</li>
+            <li class="list-item" onclick="handleListItemClick()">Button</li>
+            <li class="list-item" onclick="handleListItemClick()">Cards</li>
+            <li class="list-item" onclick="handleListItemClick()">Input</li>
+            <li class="list-item" onclick="handleListItemClick()">Image</li>
```
Comment (partha8) [inline Doc.html/cards.html]: In one review session, gaurav mentioned that we should not use handlers like onClick and more, though I forgot the reason behind it, but I stick to it. Have function and function calls in a separate js file.

Comment (abhisheksahani-nova) [reply Doc.html/cards.html]: Ohh yess I will remove that, actually its not needed.

Comment (abhisheksahani-nova) [reply Doc.html/cards.html]: Done the changes

File: Doc.html/cards.html [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+
+                <div class="card-footer">
+                  <button class="card-btn">READ</button>
+                  <button class="card-btn">BOOKMARK</button>
+                  <img
+                    class="heart-icon"
+                    src="https://img.icons8.com/fluency-systems-regular/50/000000/like--v1.png"
+                  />
+                  <img src="../Components/icons/share.png" alt="share icon" />
+                  <img src="../Components/icons/more.png" alt="more icon" />
```
Comment (partha8) [inline Doc.html/cards.html:85]: instead of having png, you can use resources like https://boxicons.com/ for icons.

**Review by partha8 — APPROVED** [review]

File: Doc.html/cards.html (hunk 2 above)
Comment (abhisheksahani-nova) [reply Doc.html/cards.html:85]: Thanks for suggesting i will use this from now on

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Looks Good

**Review by Dev-AkshatGupta — COMMENTED** [review]
Everything seems fine Abhishek just a small thing is to put description in Pr and also Please try to provide a preview link easier to find other mistakes too.

**Comment by abhisheksahani-nova** [comment]
> 



> Everything seems fine Abhishek just a small thing is to put description in Pr and also Please try to provide a preview link easier to find other mistakes too.

Sure I will do that from next time onwards

**Pull request merged**